docker/ci/dockerfiles/current/build.centos7.opensearch.x64.arm64.dockerfile

```diff
@@ -96,7 +96,7 @@ RUN pip3 install cmake==3.23.3
 # The setup part is partially based on Austin Dewey's article:
 # https://austindewey.com/2019/03/26/enabling-software-collections-binaries-on-a-docker-image/
 RUN yum install -y centos-release-scl && yum install -y devtoolset-7 && yum clean all && \
-    echo "source scl_source enable devtoolset-7" > /etc/profile.d/scl_devtoolset8.sh
+    echo "source scl_source enable devtoolset-7" > /etc/profile.d/scl_devtoolset7.sh
```

Comment:
It was meant to be set7 not set8, long time typo, see the source is actually `devtoolset-7`, doesnt change the functionality since it is just a script in profile.d tho.